plugin/src/main/kotlin/tanvd/grazi/ide/fus/GraziFUCounterCollector.kt

```diff
@@ -0,0 +1,27 @@
+package tanvd.grazi.ide.fus
+
+import com.intellij.internal.statistic.eventLog.FeatureUsageData
+import com.intellij.internal.statistic.service.fus.collectors.FUCounterUsageLogger
+import tanvd.grazi.grammar.Typo
+import tanvd.grazi.language.Lang
+
+object GraziFUCounterCollector {
+    private fun log(eventId: String, body: FeatureUsageData.() -> Unit) = FUCounterUsageLogger.getInstance()
+            .logEvent("grazi", eventId, FeatureUsageData().apply(body))
+
+    fun logLanguageDetectionResult(lang: Lang?) = log("detection") {
```

Comment:
For counter events, event id should be a verb. For instance, `language.detected`, `typo.found`, `quickfix.applied`